.github/workflows/release_linux.yml

```diff
@@ -0,0 +1,57 @@
+name: LinuxRelease
+
+on:
+  push:
+    branches: [ master, rel-*]
+  # test this PR for now, remove it before the merge
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
```

Comment:
Is there a plan to add 3.9?

Reply:
Since it's a patch for ONNX 1.8, we will stick the same supported version as ONNX 1.8. I think Python 3.9 would probably be added in ONNX 1.9. Thanks.